bio-formats/downloads/index.html

```diff
@@ -6,7 +6,7 @@
         <div class="callout large primary" id="bg-image-bio-formats">
             <div class="row column text-center">
                 <h1>Bio-Formats {{ site.bf.version }} Downloads</h1>
-                <a href="{{ site.baseurl }}/2021/03/24/bio-formats-6-6-1.html" class="large btn-red button sites-button" style="text-shadow: none;">Read the Release Announcement</a>
+                <a href="{{ site.baseurl }}/2021/08/23/bio-formats-6-7-0.html" class="large btn-red button sites-button" style="text-shadow: none;">Read the Release Announcement</a>
```

Comment:
Since https://github.com/ome/www.openmicroscopy.org/blob/5b3202b2a3cf5bf89627abc2a00181200331cf06/index.html#L74 includes a `Latest release`, this place could also be updated with a link to the announcement